tmdbv3api/objs/configuration.py

```diff
@@ -3,10 +3,47 @@
 
 
 class Configuration(TMDb):
-    _urls = {"info": "/configuration"}
+    _urls = {
+        "info": "/configuration",
+        "countries": "/configuration/countries",
+        "jobs": "/configuration/jobs",
+        "languages": "/configuration/languages",
+        "primary_translations": "/configuration/primary_translations",
+        "timezones": "/configuration/timezones"
+        }
 
     def info(self):
         """
         Get the system wide configuration info.
         """
         return AsObj(**self._call(self._urls["info"], ""))
+
+    def countries(self):
+        """
+        Get the system wide configuration info.
```

Comment:
We should add the description outlined in the TMDb documentation for this method and the others below, i.e: https://developers.themoviedb.org/3/configuration/get-languages

Reply:
Ouch! I forgot to replace each one.